packages/backend-app-api/src/wiring/createInitializationLogger.ts

```diff
@@ -29,6 +29,12 @@ export function createInitializationLogger(
   onPluginStarted(pluginId: string): void;
   onPluginFailed(pluginId: string, error: Error): void;
   onPermittedPluginFailure(pluginId: string, error: Error): void;
+  onPluginModuleFailed(pluginId: string, moduleId: string, error: Error): void;
```

Comment:
same as above, but didnt know if this should be handled independently like this, or if it should overload the existing one with an optional moduleId.